include/oneapi/dpl/internal/random_impl/uniform_real_distribution.h

```diff
@@ -141,13 +141,12 @@ class uniform_real_distribution
             __res[__i] = static_cast<scalar_type>(__engine_output[__i]);
 
         __res = ((__res - __engine.min()) / (1 + static_cast<scalar_type>(__engine.max() - __engine.min()))) *
-                    (__params.second - __params.first) +
```

Comment:
Please double check why this line was removed

Reply:
thx, returned back